dev/tests/integration/testsuite/Magento/Catalog/Model/Attribute/Backend/ConsumerTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Catalog\Model\Attribute\Backend;
+
+use Magento\AsynchronousOperations\Api\Data\OperationInterfaceFactory;
+use PHPUnit\Framework\TestCase;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\AsynchronousOperations\Model\BulkManagement;
+use Magento\AsynchronousOperations\Model\BulkStatus;
+use Magento\AsynchronousOperations\Api\Data\OperationInterface;
+use Magento\Framework\MessageQueue\BulkPublisherInterface;
+
+class ConsumerTest extends TestCase
+{
+    const BULK_UUID = '5a12c1bd-a8b5-41d4-8c00-3f5bcaa6d3c8';
+
+    /**
+     * @var \Magento\Catalog\Model\Attribute\Backend\Consumer
+     */
+    private $model;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject
+     */
+    private $publisherMock;
+
+    /**
+     * @var BulkManagement
+     */
+    private $bulkManagement;
+
+    /**
+     * @var BulkStatus
+     */
+    private $bulkStatus;
+
+    /**
+     * @var ObjectManagerInterface
+     */
+    private $objectManager;
+
+    /**
+     * @var \Magento\Framework\Serialize\SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp(): void
+    {
+        $this->objectManager = Bootstrap::getObjectManager();
+        $this->publisherMock = $this->getMockForAbstractClass(BulkPublisherInterface::class);
+
+        $this->bulkManagement = $this->objectManager->create(
+            BulkManagement::class,
+            [
+                'publisher' => $this->publisherMock
+            ]
+        );
+        $this->bulkStatus = $this->objectManager->get(BulkStatus::class);
+        $catalogProductMock = $this->createMock(\Magento\Catalog\Helper\Product::class);
```

Comment:
Hi @nuzil. What do you think about using imports everywhere instead of FQN? Currently, we use both. But I believe we may make the code a little bit more readable by using the same approach for all places.

Just a minor suggestion